sats-dna/src/main/kotlin/com/sats/dna/components/SatsEmptyState.kt

```diff
@@ -70,7 +70,12 @@ fun SatsEmptyState(
         verticalArrangement = Arrangement.spacedBy(SatsTheme.spacing.m),
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        MaterialIcon(icon, contentDescription = null, Modifier.size(36.dp))
+        MaterialIcon(
+            painter = icon,
+            contentDescription = null,
+            modifier = Modifier.size(36.dp),
+            tint = SatsTheme.colors2.surfaces.primary.fg.alternate,
+        )
```

Comment:
Not sure I agree with the design system here to always use that color. Maybe something we could pass in, with a default being this one? What if we're showing the empty state on top of a Fixed background, for instance? 🤔

Reply:
You know what? Let's deal with that when we must and leave it like this for now. Less complicated. 😎